ydb/core/tx/datashard/type_serialization.cpp

```diff
@@ -11,11 +11,11 @@
 
 namespace NKikimr::NDataShard {
 
-TString DecimalToString(const std::pair<ui64, i64>& loHi) {
+TString DecimalToString(const std::pair<ui64, i64>& loHi, const NScheme::TTypeInfo& typeInfo) {
```

Comment:
NIT: I think will be nice to replace pair with int128 or uint128 or struct

Reply:
The input for this function is: `cell.AsValue<std::pair<ui64, i64>>()`.

Reply:
Yeah, I know, just think it's not nice and can be changed (probably not in this PR)